dask/dataframe/io.py

```diff
@@ -462,14 +463,18 @@ def to_hdf(df, path_or_buf, key, mode='a', append=False, complevel=0,
                  "must preserve the order of its input")
 
     # handle lock default based on wether we're writing to a single entity
+    _actual_get = get or _globals.get('get') or get_sync
```

Comment:
If we want to go there, should we also pick the multiprocessing scheduler when writing to multiple files? Then we wouldn't have to lock at all.